File: src/core/net/packets/incoming/clicking/object/FirstClickObject.java
package core.net.packets.incoming.clicking.object;

import core.game.model.entity.player.Player;
import core.game.model.entity.player.Rights;
import core.game.plugin.PluginManager;

public class FirstClickObject {
	
	public static void handleClick(Player c, int objectType, int obX, int obY) {
		c.clickObjectType = 0;
		
		if (c.getRights().equal(Rights.DEVELOPER))
		c.sendMessage("First Click Object: " + objectType + " " + objectType + " " + obX + " " + obY);
		
		switch (objectType) {
		
		default:
			PluginManager.callFunc("objectClick1_" + objectType, c, objectType,
					obX, obY);
			break;
		}
	}
}

File: Data/plugin/scripts/first_click_object.py
__author__ = '7Winds'
from core.game.plugin import PluginManager
from core.game.model.entity.player import Player
from core import Configuration

def objectClick1_1276(c, objectType, obX, obY):
    c.sendMessage("You clicked a tree")
	
def objectClick1_2213(c, objectType, obX, obY):
	c.getActionSender().openUpBank()
	
def objectClick1_9398(c, objectType, obX, obY):
	c.getActionSender().sendString("The Bank of " + Configuration.SERVER_NAME + " - Deposit Box", 7421)
	c.getActionSender().sendFrame248(4465, 197)
	c.getInventory().resetItems(7423)